src/tests/ftest/util/server_utils.py

```diff
@@ -504,29 +517,53 @@ def kill(self):
 
     def clean_files(self):
         """Clean the tmpfs on the servers."""
+        scm_mount = self.runner.job.yaml_params.server_params[-1].scm_mount
+        scm_list = self.runner.job.yaml_params.server_params[-1].scm_list.value
         clean_cmds = [
             "find /mnt/daos -mindepth 1 -maxdepth 1 -print0 | xargs -0r rm -rf"
         ]
-
-        if self.runner.job.yaml_params.is_nvme() or \
-           self.runner.job.yaml_params.is_scm():
-            clean_cmds.append("sudo rm -rf /mnt/daos; sudo umount /mnt/daos")
-
-        self.log.info("Cleanup of /mnt/daos directory.")
+        if self.runner.job.yaml_params.is_nvme():
+            clean_cmds.append("sudo rm -rf {0};  \
+                               sudo umount {0}".format(scm_mount))
+        # scm_mount can be /mnt/daos0 or /mnt/daos1 for two daos_server
+        # instances. Presently, not supported in DAOS. The for loop needs
+        # to be updated in future to handle it. Single instance pmem
+        # device should work now.
+        if self.runner.job.yaml_params.is_scm():
+            for value in scm_list:
+                clean_cmds.append("sudo umount {}; \
```

Comment:
It seems like in the case where there are multiple values in `scm_list`, this loop is going to `umount` the same mount point in each iteration.  Is that really what should be happening?

Reply:
At some point scm_mount should be a list value... Each scm_list entry will correspond to one scm_mount. Presently, we don't support multiple scm_mounts. When we support it scm_list and scm_mount will have same amount of entries. Presently, only one can be used and I feel it should be ok.

Reply:
A comment in the YAML file explaining that lists of `scm_list` longer than 1 are not (yet) supported, would be useful.  That will prevent somebody from having to go on an easter egg hunt trying to figure out why the devices they added to the list are not being used.

Reply:
Agree.